src/triage/component/audition/__init__.py

```diff
@@ -341,3 +347,41 @@ def write_tyra_config(self, write_path):
         logging.info('Writing final model group ids to export to Tyra')
         with smart_open(write_path, 'w') as f:
             yaml.dump({'selection_rule_model_groups': self.selection_rule_model_group_ids}, f)
+
+
+class Audition(object):
+    def __init__(self, config_dict, db_engine, directory=None):
+        self.dir = directory
+        self.config = config_dict
+        self.db_engine = db_engine
+
+    def run(self):
+        pre_aud = PreAudition(self.db_engine)
+        model_group_ids = pre_aud.get_model_groups(self.config['model_groups']['query'])
+        query_end_times = self.config['time_stamps']['query'].format(', '.join(map(str, model_group_ids)))
+        end_times = pre_aud.get_train_end_times(query=query_end_times)
+
+        aud = Auditioner(
```

Comment:
Given that we're adding another layer on top of `Auditioner`, I wonder if another name for it would be more clear; I had kind of named it to be the public-facing interface of the module, but now are creating another thing that is supposed to serve as that. Having `Audition` and `Auditioner` is a bit confusing.

It's something that you would call iteratively, as opposed to a one-and-done like this new class. It's what you might use from a notebook, as opposed to the command-line or automatically in a script (like this new class).  I still don't have a good name suggestion. But if you have one in mind, this could be a good time to rename it.

Reply:
`Stage`? :stuck_out_tongue: 

Nah not really. No idea, myself.